packages/dynamic-pages/src/dynamicPages/generateFiles.js

```diff
@@ -27,8 +38,12 @@ export default new class GenerateFiles {
         // "the request of a dependency is an expression"
         // see: https://github.com/webpack/webpack/issues/196
         //      https://github.com/webpack/webpack/issues/198
-        // eslint-disable-next-line global-require
         this.dynamicRender = require('' + dynamicRenderFile).default;
```

Comment:
The main reason this is here is to have a default way to use dynamic-pages (which is synchronous) that does't break compatibility with the older version.